velox/docs/functions/spark/misc.rst

```diff
@@ -2,6 +2,11 @@
 Miscellaneous Functions
 ====================================
 
+.. spark:function:: at_least_n_non_nulls(n, value1, value2, ..., valueN) -> bool
+
+    Returns true if there are at least ``n`` non-null and non-NaN values, 
+    or false otherwise.
```

Comment:
Would you document the behavior for n being <= 0?

Reply:
Sure.